src/umpire/util/Macros.hpp

```diff
@@ -87,12 +88,14 @@
 
 #define UMPIRE_USE_VAR(x) static_cast<void>(x)
 
-
 #define UMPIRE_ERROR( msg )                                        \
 {                                                                  \
-  UMPIRE_LOG(Error, msg);                                          \
+  umpire::util::Backtrace backtrace;                               \
```

Comment:
Just to confirm, we now _always_ get a backtrace on error?

Reply:
Yes.  Except on platforms it is not supported (like Umpire).

Reply:
I assume you mean "like Windows".

Reply:
lol - oops. yes, like Windows

Reply:
@davidbeckingsale, is this resolved now?

Reply:
Yup.